boards/seeed/wio_wm1110_dev_board/board.yml

```
@@ -0,0 +1,10 @@
+# Copyright (c) 2025 Ishraq Ibne Ashraf <ishraq.i.ashraf@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+board:
+  name: wio_wm1110_dev_board
+  full_name: Wio-WM1110 Dev Board
```

```suggestion
  name: wio_wm1110_dev_kit
  full_name: Wio-WM1110 Dev Kit
```

This is the official name as far as I can tell. Rename everywhere, including folder names, image files, etc.